### PR TITLE
[opt](cloud) reduce cache hotspot table write amplification

### PR DESCRIPTION
**Pull request by freemandealer**
1. batch insert cloud_cache_hotspot in FE
2. enlarge polling interval in FE
3. shrink bucket num to 1 for cloud_cache_hotspot table
4. ignore stable statistics only catch the dynamic in BE

### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [x] Other reason (it's an opt, no behaviour has been changed)

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by Thearas** [comment]
Thank you for your contribution to Apache Doris.
Don't know what should be done next? See [How to process your PR](https://cwiki.apache.org/confluence/display/DORIS/How+to+process+your+PR).

Please clearly describe your PR:
1. What problem was fixed (it's best to include specific error reporting information). How it was fixed.
2. Which behaviors were modified. What was the previous behavior, what is it now, why was it modified, and what possible impacts might there be.
3. What features were added. Why was this function added?
4. Which code was refactored and why was this part of the code refactored?
5. Which functions were optimized and what is the difference before and after the optimization?

File: be/src/cloud/cloud_tablet_hotspot.cpp [hunk 1]
```diff
@@ -112,7 +99,10 @@ void TabletHotspot::get_top_n_hot_partition(std::vector<THotTableMessage>* hot_t
             [=, &return_partitions](
                     const std::unordered_map<TabletHotspotMapKey,
                                              std::unordered_map<int64_t, TabletHotspotMapValue>,
-                                             MapKeyHash>& hot_partition) {
+                                             MapKeyHash>& hot_partition,
```
Comment (gavinchou) [inline be/src/cloud/cloud_tablet_hotspot.cpp:102]: refactor to remove this abused lambda

**Comment by freemandealer** [comment]
run buildall

**Comment by freemandealer** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**